pkgs/tools/backup/rsbep/default.nix

```diff
@@ -18,16 +20,22 @@ stdenv.mkDerivation {
     mv rsbep_chopper $libexecDir
 
     # Fix store dependencies in scripts
-    path="export PATH=$out/bin:$libexecDir:${lib.makeBinPath [ coreutils gnused gawk ]}"
+    path="export PATH=$out/bin:$libexecDir:${lib.makeBinPath [ coreutils gawk ]}"
     sed -i "2i$path" freeze.sh
     sed -i "2i$path" melt.sh
 
-    substituteInPlace freeze.sh --replace /bin/ls ls
-
     # Remove unneded binary
     rm poorZFS.py
   '';
 
+  passthru.tests = runCommand "test-rsbep" {} ''
```

Comment:
Why don't we run this as `installCheckPhase`?

Reply:
Fixed, thanks!